web/crux/src/shared/const.ts

```diff
@@ -11,10 +11,11 @@ export const VERSIONLESS_PROJECT_VERSION_NAME = 'rolling'
 
 const DAY_IN_MILLIS = 24 * 60 * 60 * 1000
 export const TEAM_INVITATION_EXPIRATION = 7 * DAY_IN_MILLIS
-export const JWT_EXPIRATION = 10 * 60 * 1000 // 10 minutes
+export const JWT_EXPIRATION_MILLIS = 10 * 60 * 1000 // 10 minutes
 export const CONTAINER_DELETE_TIMEOUT = 1000 // millis
 
 export const DEFAULT_CONTAINER_LOG_TAIL = 40
+export const MINIMUM_AGENT_VERSION = '0.7.0'
```

Comment:
We have to always update this with committing new versions?

Reply:
...by the way, where is the usage for this const?

Reply:
This is not ready yet. I will implement that in a new ticket.